DigitalOffice/Tests/UserServiceTests/RepositoriesTests/UserRepositoryTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using System.Collections.ObjectModel;
+using System.Security.Cryptography;
+using System.Text;
+using UserService.Database;
+using UserService.Database.Entities;
+using UserService.Repositories;
+using UserService.Repositories.Interfaces;
+
+namespace UserServiceUnitTests.RepositoriesTests
+{
+    class UserRepositoryTests
+    { 
+        private IUserRepository repository;
+        private DbContextOptions<UserServiceDbContext> dbOptionsRegisterUser;
+
+        [SetUp]
+        public void Initialize()
+        {     
+            dbOptionsRegisterUser = new DbContextOptionsBuilder<UserServiceDbContext>()
+                .UseInMemoryDatabase(databaseName: "RegsterUserTest")
```

Comment:
Fix mistake to "RegisterUserTest", please.

Reply:
fixed